openupgrade_scripts/scripts/event_booth_sale/16.0.1.1/upgrade_analysis.txt

```diff
@@ -4,6 +4,9 @@ event_booth_sale / event.booth              / price (float)                 : is
 event_booth_sale / event.type.booth         / price (float)                 : is now stored
 event_booth_sale / product.template         / detailed_type (False)         : selection_keys is now '['consu', 'course', 'event', 'event_booth', 'product', 'service']' ('['consu', 'event', 'event_booth', 'gift', 'product', 'service']')
 ---XML records in module 'event_booth_sale'---
+event.booth.category: event_booth.event_booth_category_premium (noupdate) (noupdate switched)
+event.booth.category: event_booth.event_booth_category_standard (noupdate) (noupdate switched)
+event.booth.category: event_booth.event_booth_category_vip (noupdate) (noupdate switched)
```

Comment:
I see no changes between : 
https://github.com/odoo/odoo/blob/15.0/addons/event_booth_sale/data/event_booth_category_data.xml
https://github.com/odoo/odoo/blob/16.0/addons/event_booth_sale/data/event_booth_category_data.xml

Are you sure you use head version for version 15 and version 16 ?

Reply:
You are correct, i don't know why it has that, I use the head version only lack a few commit but i don't thing that is a problem :(

Reply:
Hum. That's annoying. did you used a fresh DB also ?

Reply:
Yes, i did, brand new DB. I just found a commit that they have changed to `noupdate` in orignal data, see `https://github.com/odoo/odoo/commit/67b149fc3a8dffea37de59c9579d552c559b11f7` but it shouldn't be like this right, maybe i'll delete it anyway :(

Reply:
delete what ? commit ?

Reply:
No i mean delete those 3 lines
`event.booth.category: event_booth.event_booth_category_premium (noupdate) (noupdate switched)
event.booth.category: event_booth.event_booth_category_standard (noupdate) (noupdate switched)
event.booth.category: event_booth.event_booth_category_vip (noupdate) (noupdate switched)`
Or let's me rerun again. Actually i just want to update some of major module like `account` but why not run all modules right :v

Reply:
> No i mean delete those 3 lines

OK. Well, I'd like to understand why there 3 false positive are here. Otherwise, we have a risk to add other false positive. we really have to make sure that these analysis files are correct. that's what the developers work on.

Reply:
Ok i understood, let's rerun again and see what happen, thank for spending your time to have a look .

Reply:
Just make not sure i did not skip any step, i follow step by step in this link https://oca.github.io/OpenUpgrade/analyse.html , what possibly i can miss here :((

Reply:
The problem here is that this data is only adding more values to the existing record that was defined in the module `event_booth`:

https://github.com/odoo/odoo/blob/aa90123b666784aea039ebb3c1febf0be81f8362/addons/event_booth/data/event_booth_category_data.xml#L4C77-L4C77

and if that definition, the record is noupdate.

This is then a problem of the `upgrade_analysis` module. I'm going to check if I can fix it, although the expert here is @MiquelRForgeFlow

Reply:
It seems this query has not been executed in 15.0:

https://github.com/OCA/server-tools/blob/cd2f828fed05a65aae0dc18eef26bc6f2eb31d39/upgrade_analysis/wizards/upgrade_generate_record_wizard.py#L95

I got my session kicked out though, so maybe it's because one of the flushes overwrites these values. @StefanRijnhart @hbrunn you may know what is happening here.

I'm going to execute manually the code and perform the analysis in that condition.